modules/appeals_api/config/schemas/10182.json

```diff
@@ -46,18 +46,50 @@
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
-        "representativesName": { "type": "string", "maxLength": 120 },
-        "address": { "$ref": "#/definitions/nodCreateAddress" }
+        "address":             { "$ref": "#/definitions/nodCreateAddress" },
+        "phone":               { "$ref": "#/definitions/nodCreatePhone" },
+        "emailAddressText":    { "$ref": "#/definitions/nodCreateEmail" },
+        "representativesName": { "type": "string", "maxLength": 120 }
       },
-      "required": [ "homeless", "address" ]
+      "required": [ "homeless", "phone", "emailAddressText" ]
     },
 
 
     "nodCreateAddress": {
       "type": "object",
-      "properties": { "zipCode5": { "type": "string", "pattern": "^[0-9]{5}$", "minLength": 5, "maxLength": 5 } },
       "additionalProperties": false,
-      "required": [ "zipCode5" ]
+      "properties": {
+        "addressLine1":            { "type": "string" },
+        "addressLine2":            { "type": "string" },
+        "addressLine3":            { "type": "string" },
+        "city":                    { "type": "string" },
+        "stateCode":               { "$ref": "#/definitions/nodCreateStateCode" },
+        "countryName":             { "type": "string" },
+        "zipCode5":                { "type": "string", "pattern": "^[0-9]{5}$" },
+        "internationalPostalCode": { "type": "string" }
+      },
+      "required": [ "addressLine1", "city", "countryName", "zipCode5" ]
+    },
+
+
+    "nodCreatePhone": {
+      "$comment": "the phone fields must not exceed 25 chars, when concatenated",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "countryCode":     { "type": "string", "pattern": "^[0-9]+$" },
+        "areaCode":        { "type": "string", "pattern": "^[2-9][0-9]{2}$" },
```

Comment:
Add `minLength` & `maxLength` limits to any of these address parts where the regex defines a length.

Reply:
In looking into this, I noticed that Puerto Rico' country code has a hyphen in it `1-787`, but our regex accepts . Is this something we need to consider? But! It appears that we accept up to 14 characters for a phone number, so that would account for someone entering `17873136612345`, for example. Thoughts?

Reply:
Hyphens shown in phone numbers are for readability - they aren't a requirement for input. Additionally, for Puerto Rico in particular, its country code is `1` while its area codes are `787` or `939`, so a full phone number would be something like `17875551234`.

Similarly, if you were trying to call Bermuda, which still resides in the `1` country code, a phone number for it would be `14415551234`.

Germany, however, has `49` as its country code along with a number of different area & specialty codes to where some numbers going into the Kelley Barracks, for instance, could have number similar to:
```countryCode: 49
areaCode: 711
phoneNumber: 5551234```

Reply:
Ok, I thought maybe they couldn't be separated given the resource I was looking at. I wasn't worried about the hyphen though, but number of characters. As long as we're certain country codes will always only be one digit, we're good.

Reply:
We are not certain, thus my rant regarding the German country code ;) From what I can tell the max for a country code is 3 digits, but I'm not incredibly well versed in non-US phone number validation, so let's take off the maxLength limit and [add a note to the phone number description](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/200996.json#L128) regarding max length of the combined phone number.

Reply:
It appears there is a standard for maximum phone number length: 15 digits. I provide more info in a comment below in the schema file